x/stakeibc/keeper/unbonding.go

```diff
@@ -67,17 +67,13 @@ func (k Keeper) GetQueuedHostZoneUnbondingRecords(
 }
 
 // Loops through each epoch unbonding record and returns the total number of native tokens
-// that should be unbonded, and the total number of stTokens that should be burned
-func (k Keeper) GetTotalUnbondAmount(
-	hostZoneUnbondingRecords map[uint64]recordstypes.HostZoneUnbonding,
-) (totalNativeAmount sdkmath.Int, totalStAmount sdkmath.Int) {
-	totalNativeAmount = sdk.ZeroInt()
-	totalStAmount = sdk.ZeroInt()
+// that should be unbonded
+func (k Keeper) GetTotalUnbondAmount(hostZoneUnbondingRecords map[uint64]recordstypes.HostZoneUnbonding) sdkmath.Int {
+	totalNativeAmount := sdk.ZeroInt()
 	for _, hostZoneRecord := range hostZoneUnbondingRecords {
 		totalNativeAmount = totalNativeAmount.Add(hostZoneRecord.NativeTokensToUnbond)
```

Comment:
Just to confirm, if there are partially handled records in the incoming hostZoneUnbondingRecords, this field will be only the remaining amount from previous failures -- it will already have been reduced in the callbacks which succeeded by the successfully handled amount and in order to have the RETRY_QUEUE status, all callbacks have to have resolved succeed or fail but there can't be any still outstanding... correct?

Reply:
yeah first part is right about the amount potentially being partial. RETRY_QUEUE is set as soon as a single batch fails